client/scss/utilities/_root-scope-classes.scss

```diff
@@ -40,6 +40,20 @@
   justify-content: center;
 }
 
+.flex--h-space-between {
+  justify-content: space-between;
+}
+
+.plain-button {
```

Comment:
The problem with this as it stands is that the base `btn` class is working quite hard already (and as such `btn--plain` would require much more undoing of styles). Perhaps there's a piece of work to change `btn`'s default styling in another PR.